sos/plugins/origin.py

```diff
@@ -146,6 +149,13 @@ def setup(self):
                 self.add_cmd_output("%s logs -n default %s"
                                     % (self.oc_cmd, pod))
 
+            # get a node description from the cluster ndoes
```

Comment:
It is, but its not in the most "readable" format, or separated in a way that is useful to an admin looking at the system. 

Example: 
```
{
    "kind": "Event",
    "apiVersion": "v1",
    "metadata": {
        "name": "docker-registry-7-5h9cg.146c505f7341377c",
        "namespace": "default",
        "selfLink": "/api/v1/namespaces/default/events/docker-registry-7-5h9cg.146c505f7341377c",
        "uid": "e8d4e039-664d-11e6-977b-fa163e71e8b9",
        "resourceVersion": "2574470",
        "creationTimestamp": "2016-08-19T20:45:48Z",
        "deletionTimestamp": "2016-08-22T15:40:46Z"
    },
    "involvedObject": {
        "kind": "Pod",
        "namespace": "default",
        "name": "docker-registry-7-5h9cg",
        "uid": "5574d66d-3d76-11e6-9648-fa163e71e8b9",
        "apiVersion": "v1",
        "resourceVersion": "2366618",
        "fieldPath": "spec.containers{registry}"
    },
    "reason": "Pulled",
    "message": "Container image \"registry.access.redhat.com/openshift3/ose-docker-registry:v3.2.0.46\" already present on machine",
    "source": {
        "component": "kubelet",
        "host": "node2.vault.com"
    },
    "firstTimestamp": "2016-08-19T20:45:49Z",
    "lastTimestamp": "2016-08-22T13:40:46Z",
    "count": 1125,
    "type": "Normal"
}
```
In short events are stored in "namespaces" and tied to nodes, so you would need to parse the json from the _kubernetes plugin_ to get this data viewable in a form where it is useful.